scripts/x-data-grid-premium.exports.json

```diff
@@ -660,7 +660,6 @@
   { "name": "renderEditSingleSelectCell", "kind": "Variable" },
   { "name": "renderRowReorderCell", "kind": "Variable" },
   { "name": "RowPropsOverrides", "kind": "Interface" },
-  { "name": "sanitizeFilterItemValue", "kind": "Variable" },
```

Comment:
I have removed this because I have noticed that it was unnecessarily exported [here](https://github.com/mui/mui-x/pull/15252)